From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Christian Glombek <lorbus@fedoraproject.org>
Date: Tue, 31 Jul 2018 11:11:01 +0200
Subject: [PATCH] Rename 00_menu_auto_hide.in to 01_menu_auto_hide.in

This is necessary to accommodate the fallback counting script which
needs to run before this one because the menu auto hide script sets
boot_success = 0, which will be used by the boot counting script
---
 Makefile.util.def                                          | 4 ++--
 util/grub.d/{00_menu_auto_hide.in => 01_menu_auto_hide.in} | 0
 2 files changed, 2 insertions(+), 2 deletions(-)
 rename util/grub.d/{00_menu_auto_hide.in => 01_menu_auto_hide.in} (100%)

diff --git a/Makefile.util.def b/Makefile.util.def
index 97cd8bdebd9..cba4d500198 100644
--- a/Makefile.util.def
+++ b/Makefile.util.def
@@ -449,8 +449,8 @@ script = {
 };
 
 script = {
-  name = '00_menu_auto_hide';
-  common = util/grub.d/00_menu_auto_hide.in;
+  name = '01_menu_auto_hide';
+  common = util/grub.d/01_menu_auto_hide.in;
   installdir = grubconf;
 };
 
diff --git a/util/grub.d/00_menu_auto_hide.in b/util/grub.d/01_menu_auto_hide.in
similarity index 100%
rename from util/grub.d/00_menu_auto_hide.in
rename to util/grub.d/01_menu_auto_hide.in
